openslides/users/rest_auth.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework.authentication import BaseAuthentication
+
+from openslides.config.api import config
+
+
+# TODO: Move this into the file 'auth' after fixing circular imports
```

Comment:
There should no be an import from on of these apps (utils, config, rest_framework) to our users app but only in other direction. Which import line in utils/rest_api.py causes this loop? I can't imagine this error.